Actions/ReadSecrets/ReadSecrets.ps1

```diff
@@ -113,12 +123,17 @@ try {
         }) -join ', ')"
     }
 
+    Write-Host "OUTPUTS:"
+    $outSettingsJson = $outSettings | ConvertTo-Json -Depth 99 -Compress
+    Write-Host "- SettingsJson=$outSettingsJson"
+    if ($useBase64) {
+        $outSettingsJson = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($outSettingsJson))
```

Comment:
Since at this point the `$outSettingsJson` may contain secrets, it's quite dangerous to encode it as the encoded string is in no way masked (see [here](https://github.com/freddydk/ALAppExtensions/actions/runs/5657371091/job/15327074618#step:8:100) for example).

Reply:
Adding the entire settings construct will mask the encoded settings and minimize the clutter in the logs